### ISSUE TITLE
Custom callout not showing? 

### ISSUE THREAD

**Issue by jojonarte** [lelandrichardson/react-native-maps#377]
These are my dependencies 
```
`  "dependencies": {
    "express": "^4.14.0",
    "node-googleplaces": "^0.1.2",
    "react": "15.2.0",
    "react-native": "^0.29.0",
    "react-native-maps": "^0.6.0",
    "yelp": "^1.0.1"
  }`
```

I'm trying to display the names of each businesses returned via yelp and google place api, I've not been able to let my custom callout display on the marker. Can someone help me point out where I've gotten wrong?

This is my map code
`
```
<MapView
              style={styles.map}
              region={this.state.region}
              onRegionChange={this.onRegionChange.bind(this)}
              showsUserLocation={true}
              loadingEnabled={true}
              followsUserLocation={true}
              >

              {markers.map(marker => (
                <MapView.Marker
                    coordinate={this.setCoordinate(marker)}
                    key={marker.name}
                  >

                  <MapView.Callout>
                       <CustomCallout marker={marker} />
                  </MapView.Callout>

                </MapView.Marker>
              ))}

              </MapView>
```
`

The custom callout is this

```
class CustomCallout extends React.Component{
  constructor(props){
      super(props);
      this.state= {
      }
  }

  componentDidMount(){

  }

  render(){
      var marker = this.props.marker;
      console.log(marker.name);
        return (
          <View style={{flex:1, flexDirection:'column'}}>
              <Text style={styles.textHead}>{marker.name}</Text>
              <Text>HEEH</Text>
              <Text style={styles.textHead}>{marker.name}</Text>
              <Text>HEEH</Text>
              <Text style={styles.textHead}>{marker.name}</Text>
              <Text>HEEH</Text>
              <Text style={styles.textHead}>{marker.name}</Text>
              <Text>HEEH</Text>
              <Text style={styles.textHead}>{marker.name}</Text>
              <Text>HEEH</Text>
          </View>

        )


  }

```

**Comment by jojonarte** [comment]
![screen](https://cloud.githubusercontent.com/assets/11880365/16718621/4537d35e-4754-11e6-8bef-cc2bcb8ddeca.png)

Screenshot above might help.

**Comment by KGALLET** [comment]
I'm not sure if that's the problem but I needed to put a style `<MapView.Callout style={ styles.myCallout }>` to set a width property in order to see my custom callout

**Comment by willoughby** [comment]
Had the same problem which @KGALLET comment resolved, RN 0.29

**Comment by jojonarte** [comment]
The width solved the problem :) 👍
